src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/MSBuildProject.NuGetPackages.cs

```diff
@@ -156,8 +156,9 @@ private IEnumerable<LockFileTargetLibrary> GetAllDependencies(TargetFrameworkMon
             var lockFile = LockFileUtilities.GetLockFile(LockFilePath, NuGet.Common.NullLogger.Instance);
 
             return lockFile.Targets
-                .First(t => t.TargetFramework.DotNetFrameworkName.Equals(parsedTfm.DotNetFrameworkName, StringComparison.Ordinal))
-                .Libraries;
+                .FirstOrDefault(t => t.TargetFramework.DotNetFrameworkName.Equals(parsedTfm.DotNetFrameworkName, StringComparison.Ordinal))
```

Comment:
The fix for this is actually a little different - I'm planning on implementing that today

Reply:
Can I leave this as is for this PR, or do you want it removed from here?

Reply:
It'll be a merge conflict most likely, so if you could remove it, that would be ideal

Reply:
Removed